src/coreclr/jit/utils.cpp

```diff
@@ -1328,19 +1328,10 @@ void hexDump(FILE* dmpf, const char* name, BYTE* addr, size_t size)
 
     assert(addr);
 
-    fprintf(dmpf, "Hex dump of %s:\n", name);
-
     for (unsigned i = 0; i < size; i++)
     {
-        if ((i % 16) == 0)
```

Comment:
I am assuming we don't want this anymore and we will never print the offsets?

Reply:
Yea, we will just print the hex-code as is. The disassembly tool we will use can put in the offsets when decoding I believe.